webpack/vendorRegistry.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  'axios',
+  'react',
+  'react-dom',
+  'lodash',
```

Comment:
@pdw207 - I find lodash pretty handy for some of the more complex operations not handled by ES6 out of the box, so I, for one, wouldn't mind keeping it in unless anyone is opposed